### DISCUSSION TITLE
How to delete identical files/folders in one big swoop, only keeping different or unique data

### DISCUSSION THREAD

**Discussion by ricklmartin** [WinMerge/winmerge#2240] [Q&A]
I had to restore about 200 GB of data to a seperate folder for comparison with the current data. I want to keep that restored data, but only files/folders that are different or unique. I don't want all the extra identical data in this restored set, which is already in the current set. But for the life of me, I cannot find the answers, nor figure it out without having to manually go through deleting everything. Is there an automated way to do this? I thought changing the view to ONLY show Identical, then select all to delete, but that deletes everything regardless of the view setting.

**Comment by sdottaka** [comment]
> I thought changing the view to ONLY show Identical, then select all to delete, but that deletes everything regardless of the view setting.

If you select all while Tree Mode is enabled, subfolders will also be selected, so the subfolders and all items in them will be deleted.

To avoid including subfolders, how about unchecking the View → Tree Mode menu item, sorting by the Comparison result column, and selecting and deleting only the identical items?

**Reply by gordonfreeman01** [reply]
This is a good idea for deleting individual files but in my use case I have a hierarchy where there might be one shared folder, one different sub-folder and a file in that folder so to delete the different sub-folder and file within it that method wouldn't work.
